package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "private": true,
+  "scripts": {
+    "postinstall": "pulp dep install",
+    "build-example": "pulp dep link && cd example && pulp dep link purescript-dom-keyboard && pulp build",
+    "build": "pulp build"
```

Comment:
This `bower link` method for testing subprojects is what we do on Halogen too, it's not always necessary, but when not using gulp it's probably the easiest way. If it was a test rather than an example we could just use the `test/` dir and `pulp test` instead.